docs/docs/platform/workflows/create-workflow.md

```diff
@@ -0,0 +1,535 @@
+---
+sidebar_position: 1
+sidebar_label: Create Workflow
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Create Workflow
+
+We have an elegant drag and drop interface to create a new `workfklow`. User can wither create new workflow from scratch, by selecting existing workflow from our workflow store or programmatically using API/SDK. Click on the `Workflows` option in the left sidebar to access all workflows. Workflows page lists all exisiting workflows ordered by last creation date.
```

Comment:
```suggestion
We have an elegant drag and drop interface to create a new `workflow`. User can either create a new workflow from scratch, select an existing workflow from our Template Store or create one programmatically using API/SDK. Click on the `Workflows` option in the left sidebar to access all workflows. Workflows page lists all existing workflows ordered by last creation date.
```
Note, that we call it Template Store , also it would be nice to have a link to template store